README.md

```diff
@@ -58,6 +60,24 @@ Migrate databases:
 
     (capstone)$ fab migrate
 
+Docker Setup
+------------
+
+We have initial support for local development via `docker compose`. Docker setup looks like this:
+
+    $ docker-compose up &
+    $ docker-compose exec db psql --user=postgres -c "CREATE DATABASE capstone;"
+    $ docker-compose exec db psql --user=postgres -c "CREATE DATABASE capapi;"
+    $ docker-compose exec web fab init_db
+    $ docker-compose exec web fab load_test_data
+    
+Capstone should now be running at 127.0.0.1:8000.
+
+These commands can be shorter with something like this in .bash_profile:
+
+    alias d="docker-compose exec"
+    alias dfab="d web fab"
```

Comment:
And accolades from the peanut gallery: nice aliases! these are improvements on the ones I've been using!